cmd/manager/main.go

```diff
@@ -125,6 +127,15 @@ func main() {
 		os.Exit(1)
 	}
 
+	TrackingStatus := strings.ToUpper(os.Getenv("TRACKING_STATUS"))
+	// Check if the TrackingStatus env has been negated or not
+	if TrackingStatus == "TRUE" {
```

Comment:
This function should run by default, And if `ANALYTIC: "false"` in env they only it will not run
Need to changes like
```
if isAnalytics != "FALSE" {
	err := analytics.TriggerAnalytics()
	if err != nil {
		log.Error(err, "")
	}
}
```